.github/CODEOWNERS

```diff
@@ -1 +1 @@
-.github/CODEOWNERS @sonarsource/sonarcloud
+.github/CODEOWNERS @sonarsource/analysis-experience
```

Comment:
![image](https://github.com/SonarSource/sonar-scanner-vsts/assets/31401273/a7f4be62-8865-48f3-8b11-99665fbaa8bf)

____

Shouldnt that be
```suggestion
.github/CODEOWNERS @sonarsource/analysis-experience-squad
```

as per [our team ID](https://github.com/orgs/SonarSource/teams/analysis-experience-squad)?
?

Reply:
Yes, it is `@sonarsource/analysis-experience-squad`